tests/test_api_skeleton.py

```diff
@@ -448,19 +453,31 @@ def test_pypi_section_order_preserved():
 
 @pytest.mark.slow
 @pytest.mark.flaky(rerun=5, reruns_delay=2)
-@pytest.mark.skipif(not external.find_executable("shellcheck"), reason="requires shellcheck >=0.7.0")
+@pytest.mark.skipif(on_win, reason="shellcheck is only available on Windows")
```

Comment:
I didn't know shellcheck is only on Windows, shouldn't that be:
```suggestion
@pytest.mark.skipif(on_win, reason="shellcheck is not available on Windows")
```

Reply:
🤦🏼‍♂️

Reply:
🤗